dspace-api/src/main/java/org/dspace/storage/rdbms/DatabaseManager.java

```diff
@@ -688,9 +673,13 @@ public static void freeConnection(Connection c)
      * @return The newly created row
      * @throws SQLException
      */
+    public static TableRow row(Context context, String table) throws SQLException
+    {
+        return new TableRow(canonicalize(table), getColumnNames(context, table));
+    }
     public static TableRow row(String table) throws SQLException
     {
-        return new TableRow(canonicalize(table), getColumnNames(table));
+        return new TableRow(canonicalize(table), getColumnNames(null, table));
```

Comment:
You create a `getColumnNames(table)` method later on in this class. This change is unnecessary, as it should just use that other method.